fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/FedoraTypesUtils.java

```diff
@@ -107,12 +107,16 @@
     });
 
     /**
+     * Check if a property is a reference property.
```

Comment:
This comment seems like a copy/paste error.
Also, this function `isSuppressedProperty` should be `private`.